src/components/SignInSignUp.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import SignInForm from '../components/SignInForm';
+import SignUpForm from '../components/SignUpForm';
+import Logo from '../components/ui/Logo';
+import { Tabs, Tab } from '../components/ui/Tabs';
+import Card from '../components/ui/Card';
+
+import '../styles/components/_SignInSignUp.scss';
+
+class SignInSignUp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSignIn = this.handleSignIn.bind(this);
+    this.handleSignUp = this.handleSignUp.bind(this);
+  }
+
+  handleSignIn(data) {
+    console.log('SIGN IN');
+  }
+
+  handleSignUp(data) {
+    console.log('SIGN UP');
```

Comment:
Yes. We will put the actual code there instead of the `console.log`. Despite the absence of the actual code, this stub is a good "checkpoint" to be pulled in so that both of us have access to it and that any of us could build upon it.